src/reports/ChartReportPage.jsx

```diff
@@ -161,7 +164,14 @@ const ChartReportPage = () => {
                 formatter={(value, key) => [value, positionAttributes[key]?.name || key]}
                 labelFormatter={(value) => formatTime(value, 'seconds')}
               />
-              <Line type="monotone" dataKey={type} stroke={theme.palette.primary.main} />
+              <Brush
+                dataKey={timeType}
+                height={30}
+                stroke={theme.palette.primary.main}
+                tickFormatter={() => ''}
```

Comment:
Is there a way to hide it completely instead of just rendering empty value?

Reply:
I can't figure a way now, setting `tick={false}` doesn't work, another option ia to set the time and format with `tickFormatter={(value) => formatTime(value, 'time')}` Please do you have a suggestion?

Reply:
Let's leave it as empty for now.